### PR TITLE
updates to the wizard and canvas-view patterns

### PR DESCRIPTION
**Pull request by michael-coker**
Punctuation, formatting, and spelling updates.

### CODE REVIEW

File: pattern-library/communication/wizard/design/design.md [hunk 1]
```diff
@@ -42,50 +42,51 @@ In the example above, the back button is disabled since this shows the first sub
...
-1. **Current Sub-step** (Optional)**:** If the current main step has sub-steps, the name of the sub-step appears next to it at the top of the page. If the main and sub-step names are long enough that [truncation](https://www.patternfly.org/styles/terminology-and-wording/#truncation) is required, the sub-step name should be truncated before the main step name unless the sub-step name is critical to filling in the form.
-1. **Steps Dropdown:** Clicking on this dropdown reveals all steps in the wizard and enables users to switch between them if applicable
+1. **Current Main Step:** The currently active main step is displayed at the top of the form along with the step number.
+1. **Current Sub-step** (optional)**:** If the current main step has sub-steps, the name of the sub-step appears next to it at the top of the page. If the main and sub-step names are long enough that [truncation](https://www.patternfly.org/styles/terminology-and-wording/#truncation) is required, the sub-step name should be truncated before the main step name unless the sub-step name is critical to filling in the form.
```
Comment (LHinson) [inline pattern-library/communication/wizard/design/design.md]: This is nit-picky but when the ":" proceeds "(option)", it's not bold

File: pattern-library/communication/wizard/design/design.md [hunk 2]
```diff
@@ -42,50 +42,51 @@ In the example above, the back button is disabled since this shows the first sub
...
+1. **Steps Dropdown:** Clicking on this dropdown reveals all steps in the wizard and enables users to switch between them if applicable.
 1. **Button Bar:** Wizard actions are available on the button bar, which is fixed at the bottom of the page for embedded wizards and modals wizards except for in smaller screen sizes, which requires the user to scroll the page for the button bar.
 
 ![Expanded responsive wizard](img/responsive-wizard-callout2.png)
 
 1. **Main Steps:** Main wizard steps are shown vertically when the steps dropdown is expanded. Clicking on a different step will display its sub-steps, or switch to it if it does not have any sub-steps.
-1. **Sub-steps for the Selected Main Step** (Optional)**:** Clicking on a sub-step will switch to that sub-step. The current sub-step is highlighted.
+1. **Sub-steps for the Selected Main Step** (optional)**:** Clicking on a sub-step will switch to that sub-step. The current sub-step is highlighted.
```
Comment (LHinson) [inline pattern-library/communication/wizard/design/design.md]: un bold the colon and all other instances in this doc

File: pattern-library/communication/wizard/design/design.md (hunk 1 above)
Comment (michael-coker) [reply pattern-library/communication/wizard/design/design.md]: Nice catch! That (and optional/Optional case discrepancies) are on some other pages, too. Mind if I stuff them in with this update?

**Comment by LHinson** [comment]
changes look good. looks like you have some conflicts though

**Comment by michael-coker** [comment]
How about now? (I haven't dealt with conflicts much, so hopefully that resolved it)

**Pull request merged**